linux_kernel_modules/mangoh/mangoh_yellow.c

```diff
@@ -457,13 +458,21 @@ static int mangoh_yellow_probe(struct platform_device* pdev)
 
  	/* Map the I2C Light Sensor */
 	dev_dbg(&pdev->dev, "mapping Light Sensor\n");
-	if (devm_gpio_request_one(&pdev->dev, CF3_GPIO36, GPIOF_DIR_IN,
-				  "CF3 opt300x gpio interrupt")) {
+	light_sensor_gpio_res = devm_gpio_request_one(
+		&pdev->dev, CF3_GPIO36, GPIOF_DIR_IN,
+		"CF3 opt300x gpio interrupt");
+	if (light_sensor_gpio_res == -EPROBE_DEFER) {
+		/*
+		 * Assume this module doesn't have a low power MCU and thus this
+		 * GPIO isn't available
+		 */
```

Comment:
Do we need to set mangoh_yellow_light_devinfo.irq to something here to mark it as unused?  Or is the structure already pre-initialized to something that indicates that?

Reply:
The default value of 0 means that there is no irq, so we don't need to explicitly initialize it.